src/main/mainEvents.ts

```diff
@@ -8,6 +8,7 @@ import { EventEmitter } from 'events';
 import type { VMBackend } from '@/backend/backend';
 import type { Settings } from '@/config/settings';
 import { RecursivePartial } from '@/utils/typeUtils';
+import { TransientSettings } from '~/store/transientSettings';
```

Comment:
`@` should be the proper alias 🙂

```suggestion
import { TransientSettings } from '@/store/transientSettings';
```